src/viur/core/utils/parse.py

```diff
@@ -2,6 +2,7 @@
 ViUR utility functions regarding parsing.
 """
 import typing as t
+from datetime import timedelta as td
```

Comment:
```suggestion
from datetime import timedelta as td
```
Please use `import datetime`.